src/compiler/crystal/macros/interpreter.cr

```diff
@@ -363,9 +363,14 @@ module Crystal
         end
 
         args = node.args.map { |arg| accept arg }
+        named_args = nil
+
+        if nargs = node.named_args
+          named_args = nargs.to_h { |arg| {arg.name, accept arg.value} }
+        end
```

Comment:
🎨 Optional! I guess we could change this to be:

```
args = node.args.map { |arg| accept arg }
named_args = node.named_args.try &.to_h { |arg| {arg.name, accept arg.value} }
```

So the `args` and `named_args` assignments are next to each other and it's clear we are mapping `node.args` and `node.named_args`.